wazimap_ng/cache.py

```diff
@@ -132,14 +132,17 @@ def subindicator_group_update(sender, instance, **kwargs):
         update_profile_cache(profile_obj)
 
 
+@receiver(post_delete, sender=Location)
 @receiver(post_save, sender=Location)
 def point_updated_location(sender, instance, **kwargs):
-    update_point_cache(instance.category)
+    for pc in instance.category.profilecategory_set.all():
+        update_point_cache(pc)
 
 
 @receiver(post_save, sender=Category)
 def point_updated_category(sender, instance, **kwargs):
-    update_point_cache(instance)
+    for pc in instance.profilecategory_set.all():
+        update_point_cache(instance)
```

Comment:
that does not look right. a `Category` does not have a `profilecategory_set` right?

but we might want to add a cache update for the `ProfileCategory` right?